pennylane_sf/remote.py

```diff
@@ -24,6 +24,7 @@
 import strawberryfields as sf
 
 # import state preparations, gates and measurements
+import xcc
```

Comment:
Will we need to add `xcc` as a dependency? Or it's okay as it ships with Strawberry Fields?

Reply:
Best to let it come via SF (since SF might want to pin XCC to a particular version).

However, there is a subtlety --- only the latest version of SF depends on XCC, so a user with an old SF version pre-installed might have an issue when installing this plugin.

A solution could be to increase the pinning of SF in the plugin `setup.py`, to the latest one?

Reply:
pinned SF to the latest in `setup.py` as suggested